examples/qdrantdb/preFilters.ts

```diff
@@ -36,19 +37,21 @@ async function main() {
     const ctx = await storageContextFromDefaults({ vectorStore: qdrantVs });
 
     console.log("Embedding documents and adding to index");
-    const index = await VectorStoreIndex.fromDocuments(docs, {
-      storageContext: ctx,
-      serviceContext: serviceContextFromDefaults({
-        callbackManager: new CallbackManager({
-          onRetrieve: (data) => {
-            console.log(
-              "The retrieved nodes are:",
-              data.nodes.map((node) => node.node.getContent(MetadataMode.NONE)),
-            );
-          },
-        }),
+    const index = await runWithCallbackManager(
+      new CallbackManager({
```

Comment:
@himself65 as we're changing this anyhow, how about simplifying it to:
```suggestion
const index = await runWithCallbacks({
```